op-e2e/actions/sync_test.go

```diff
@@ -807,27 +782,146 @@ func TestELSyncTransitionstoCL(gt *testing.T) {
 	// This was failing prior to PR 9661 because op-node would attempt to immediately insert blocks into the EL inside the engine queue. op-geth
 	// would not be able to fetch the second range of blocks & it would wipe out the unsafe payloads queue because op-node thought that it had a
 	// higher unsafe block but op-geth did not.
-	id, err = verifier.eng.L2BlockRefByLabel(t.Ctx(), eth.Unsafe)
-	require.NoError(t, err)
-	require.Equal(t, uint64(22), id.Number)
+	VerifyBlock(t, verifier.eng, 22, eth.Unsafe)
 
 	// Create 1 more block & batch submit everything
+	BatchSubmitBlock(t, miner, sequencer, verifier, batcher, dp, 12)
+
+	// Verify that the batch submitted blocks are there now
+	VerifyBlock(t, sequencer.eng, 23, eth.Safe)
+	VerifyBlock(t, verifier.eng, 23, eth.Safe)
+}
+
+func TestELSyncTransitionsToCLSyncAfterNodeRestart(gt *testing.T) {
```

Comment:
This test and `TestForcedELSyncCLAfterNodeRestart` still seem to share a lot of common code. Is there a way to deduplicate?

Reply:
I deduplicated the tests here :+1: : https://github.com/ethereum-optimism/optimism/pull/10767/commits/5114a0ad2f5385a387fa980f3de44dfd998e651b